test/tests/multiapps/picard_postprocessor/steady_main.i

```diff
@@ -0,0 +1,103 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 5
+  ny = 5
+  parallel_type = replicated
+  uniform_refine = 1
+[]
+
+[Variables]
+  [u]
+  []
+[]
+
+[Kernels]
+  [diff]
+    type = Diffusion
+    variable = u
+  []
+  [source]
+    type = BodyForce
+    variable = u
+    value = 1
+  []
+[]
+
+[BCs]
+  [left]
+    type = PostprocessorDirichletBC
+    variable = u
+    boundary = left
+    postprocessor = 'from_sub'
+  []
+[]
+
+[Postprocessors]
+  [from_sub]
+    type = Receiver
+    default = 0
+  []
+  [to_sub]
+    type = SideAverageValue
+    variable = u
+    boundary = right
+  []
+  [average]
+    type = ElementAverageValue
+    variable = u
+  []
+[]
+
+[Executioner]
+  type = Steady
+
+  # Solve parameters
+  solve_type = PJFNK
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+  nl_abs_tol = 1e-14
+
+  # App coupling parameters
+  fixed_point_max_its = 100
+  fixed_point_rel_tol = 0.5  # pseudo transient is slow to converge
+  relaxation_factor = 0.8
+  transformed_postprocessors = 'from_sub'
+[]
+
+[Outputs]
+  csv = true
+  exodus = false
+[]
+
+[MultiApps]
+  [sub]
+    type = FullSolveMultiApp
+    app_type = MooseTestApp
+    positions = '0 0 0'
+    input_files = steady_sub.i
+    clone_master_mesh = true
+    execute_on = 'timestep_begin'
+
+    relaxation_factor = 0.8
+    transformed_postprocessors = 'from_main'
```

Comment:
Could we figure out this from the transfer block automatically?

Reply:
It does not have to be a transferred pp. It could be a pp based on a variable received

Reply:
Good to know